.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,15 +12,11 @@
 
 ## Safety Assurance
 
-- [ ] Risk label is set correctly
-- [ ] All migrations are backwards compatible and won't block deploy
-- [ ] The set of people pinged as reviewers is appropriate for the level of risk of the change
-- [ ] If QA is part of the safety story, the "Awaiting QA" label is used
-- [ ] I have confidence that this PR will not introduce a regression for the reasons below
-
-## Migrations
-
-- [ ] The migrations in this code can be safely applied first independently of the code
+### Safety story
+<!--
+Describe any other pieces to the safety story including
```

Comment:
nit: without the checkbox list here, it's odd to say "any other pieces".  Maybe a rewording like this?
```suggestion
Describe how you became confident in this change, such as
```